src/deepDiff.js

```diff
@@ -3,44 +3,68 @@ import _isFunction from 'lodash/isFunction'
 import _isObject from 'lodash/isObject'
 import _keys from 'lodash/keys'
 import _union from 'lodash/union'
+import Immutable from 'immutable';
```

Comment:
is it possible to conditionally include this? e.g.

```
let Immutable;
if (useImmutable) {
  Immutable = require('Immutable');
}
```

Unsure how you would define `useImmutable` - could pass an extra item to constructor of deepDiff and require it there, otherwise env var or something. Or would this be removed by dead-code elimination if not used?